go.mod

```diff
@@ -52,6 +54,7 @@ require (
 	go.etcd.io/etcd v3.3.13+incompatible
 	go.uber.org/atomic v1.4.0
 	go.uber.org/zap v1.10.0
+	golang.org/x/tools v0.0.0-20190524140312-2c0ae7006135
```

Comment:
show u use version instead of ``` v0.0.0-20190524140312-2c0ae7006135```

https://github.com/golang/tools/releases

Reply:
I don't use this tools. I will remove this.